src/main/kotlin/apply/application/AdministratorDtos.kt

```diff
@@ -0,0 +1,39 @@
+package apply.application
+
+import apply.domain.administrator.Administrator
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Size
+
+data class AdministratorData(
+    @field:NotBlank
+    @field:Size(min = 1, max = 20)
+    @field:Pattern(
+        regexp = "[0-9a-zA-Z가-힣]+",
+        message = "이름은 영어, 한글, 숫자만 가능합니다."
+    )
```

Comment:
c: `@Pattern` 어노테이션 하나만으로 `@NotBlank`와 `@Size`를 대체할 수 있지 않을까요? 

```suggestion
    @field:Pattern(
        regexp = "[0-9a-zA-Z가-힣]{1,20}",
        message = "이름은 영어, 한글, 숫자만 가능합니다."
    )
```

정규표현식이 맞는지 모르겠지만 이런 식으로요!

Reply:
이 부분은 조금 고민해볼 필요가 있을 것 같네요. `@size`를 명시적으로 적어주는게 읽는 사람이 더 빠르게 파악할 수 있을 것 같다는 생각이 들어서요.. (정규표현식으로 전환하는게 맞다면, 모든 문자열에 대한 `@validation` 은 정규표현식으로 다 커버할 수 있으니까요.)

추가적으로 또 든 생각이, 코틀린에서 nullable 타입을 별도로 지원해주다보니, `@NotBlank` 가 과연 필요한가? 에 대한 생각이 드네요. `@NotBlank` 는 공백과 null 두 가지를 체크해주는 걸로 알고 있는데, null은 코틀린에 의해 처리되고, 공백은 `pattern`에 의해 처리될 수 있으니..

일단은 `@NotBlank`만 제거해보고, 파랑이 추가 의견 주시면 재수정하겠습니다 ~

Reply:
차리 얘기를 들어보니 `@Size`를 명시적으로 적어주는 것도 좋은 것 같네요! 다만 다른 곳에서는 `@Patterm`만 사용하는 것 같아서 어느 방식으로든 통일하면 좋을 것 같아요!